recipes-extended/xen/xen-openxt.inc

```diff
@@ -10,7 +10,6 @@ EXTRA_OECONF_remove = " \
     --disable-ioemu-stubdom \
     --disable-pv-grub \
     --disable-xenstore-stubdom \
-    --disable-rombios \
```

Comment:
nit: an explanation in the commit message of why we want rombios would be helpful

Reply:
this is removing a remove of a disable... so I think that means that the result is that we _don't_ have rombios